### PR TITLE
Adding support for cert-manager

### PR DESCRIPTION
**Pull request by tcyran**
Adding alternative way of certificate management using cert-manager.

I didn't manage to get patch-job to work properly on on k8s v1.23, so I would like to propose possibility to use cert-manager approach as alternative.

# Tasks

- [x] Bump the chart version (`Chart.yaml` -> `version`)
- [x] JSON Schema updated (`values.schema.json`)
- [x] Update `README.md` via helm-docs
- [x] Update Artifacthub annotation (`Chart.yaml` -> `artifacthub.io/changes`, `artifacthub.io/images`)

### CODE REVIEW

File: charts/k8s-image-swapper/README.md [hunk 1]
```diff
@@ -10,7 +10,8 @@ Mirror images into your own registry and swap image references automatically.
 
 | Name | Email | Url |
 | ---- | ------ | --- |
-| estahn | enrico.stahn@gmail.com |  |
+| estahn | <enrico.stahn@gmail.com> |  |
+| tcyran | <tcyran@gmail.com> |  |
```
Comment (estahn) [inline charts/k8s-image-swapper/README.md:14]: @tcyran Typically there is a distinction between `maintainers` and `contributors`. This table is showing only maintainers, not contributors. Could you remove your name here? You will be mentioned in the release notes.

File: charts/k8s-image-swapper/values.yaml [hunk 2]
```diff
@@ -35,10 +35,12 @@ podSecurityPolicy:
 
 podAnnotations: {}
 
-podSecurityContext: {}
+podSecurityContext:
```
Comment (estahn) [inline charts/k8s-image-swapper/values.yaml:38]: Can you fix this up so it doesn't appear as change?

File: charts/k8s-image-swapper/Chart.yaml [hunk 3]
```diff
@@ -13,9 +13,11 @@ keywords:
 maintainers:
   - email: enrico.stahn@gmail.com
     name: estahn
+  - email: tcyran@gmail.com
```
Comment (estahn) [inline charts/k8s-image-swapper/Chart.yaml:16]: @tcyran Typically there is a distinction between maintainers and contributors. This table is showing only maintainers, not contributors. Could you remove your name here? You will be mentioned in the release notes.

**Review by estahn — CHANGES_REQUESTED** [review]
@tcyran Thanks for the contribution. Overall it looks good, see my comments for some minor changes.

File: charts/k8s-image-swapper/README.md (hunk 1 above)
Comment (tcyran) [reply charts/k8s-image-swapper/README.md]: Clear, removed

File: charts/k8s-image-swapper/values.yaml (hunk 2 above)
Comment (tcyran) [reply charts/k8s-image-swapper/values.yaml]: Fixed

File: charts/k8s-image-swapper/Chart.yaml (hunk 3 above)
Comment (tcyran) [reply charts/k8s-image-swapper/Chart.yaml]: Clear, removed

**Comment by kjvellajr** [comment]
@estahn any status updates on the approval for this change?

**Comment by kjvellajr** [comment]
@estahn thank you!!

**Review by estahn — APPROVED** [review]

**Pull request merged**